packages/odie-client/src/components/message/bot-message-actions.tsx

```diff
@@ -0,0 +1,116 @@
+import { MessageActions, CopyIcon, ThumbsDownIcon, ThumbsUpIcon } from '@automattic/agenttic-ui';
+import { Icon } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { check } from '@wordpress/icons';
+import { ODIE_THUMBS_DOWN_RATING_VALUE, ODIE_THUMBS_UP_RATING_VALUE } from '../../constants';
+import { useOdieAssistantContext } from '../../context';
+import { useSendOdieFeedback } from '../../data';
+import type { Message } from '../../types';
+
+const BotMessageActions = ( { message }: { message: Message } ) => {
+	const { setMessageLikedStatus, trackEvent, setChatStatus } = useOdieAssistantContext();
+	const { mutateAsync: sendOdieMessageFeedback } = useSendOdieFeedback();
+	const [ isCopied, setIsCopied ] = useState( false );
+
+	const liked = message.rating_value?.toString() === '1' || message.liked || false;
```

Comment:
```suggestion
	const liked = message.rating_value?.toString() === '1' || message.liked;
```

The first condition will handle the `false` scenario.